public-servers.txt

```diff
@@ -1,3 +1,7 @@
+1.1.1.1
```

Comment:
IPv4 and IPv6 instances for Cloudflare are already in the file.

Reply:
ah my bad, maybe because they aren't sorted, will submit an updated one